bug: Error [ERR_MODULE_NOT_FOUND]: Cannot find module node_modules/ethers/lib/utils import with NextJS

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Package Version

^0.8.0

### Current Behavior

I'm following along with a [Moralis tutorial ](https://docs.moralis.io/docs/sign-in-with-metamask) and after installing all the necessary dependencies and trying to run `npm run dev` to compile everything and run the app in browser, I get the following error:

`Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/Users/repos/fg2/node_modules/ethers/lib/utils' imported from /Users/repos/fg2/node_modules/@wagmi/core/dist/chunk-2EDVJWOA.js
Did you mean to import ethers/lib/utils.js?
    at new NodeError (node:internal/errors:371:5)
    at finalizeResolution (node:internal/modules/esm/resolve:416:11)
    at moduleResolve (node:internal/modules/esm/resolve:932:10)
    at defaultResolve (node:internal/modules/esm/resolve:1044:11)
    at ESMLoader.resolve (node:internal/modules/esm/loader:422:30)
    at ESMLoader.getModuleJob (node:internal/modules/esm/loader:222:40)
    at ModuleWrap.<anonymous> (node:internal/modules/esm/module_job:76:40)
    at link (node:internal/modules/esm/module_job:75:36) {
  code: 'ERR_MODULE_NOT_FOUND'`

I'm running `node v16.13.0`.

The ethers package is definitely installed and there is definitely a `node_modules/ethers/lib/utils`. I'm really confused why it's choking and unable to find it.

### Expected Behavior

I expect the ethers import to work. :) 

### Steps To Reproduce

I suppose you could go through the first 7 steps of the tutorial and see if you can replicate the issue.

Thank you for your help!

--- a/.changeset/wicked-ladybugs-raise.md
+++ b/.changeset/wicked-ladybugs-raise.md
@@ -1,0 +1,6 @@
+---
+'@wagmi/core': patch
+'wagmi': patch
+---
+
+Fixed ethers import path

--- a/packages/core/src/actions/ens/fetchEnsName.ts
+++ b/packages/core/src/actions/ens/fetchEnsName.ts
@@ -1,5 +1,5 @@
 import { Address } from 'abitype'
-import { getAddress } from 'ethers/lib/utils'
+import { getAddress } from 'ethers/lib/utils.js'
 
 import { getProvider } from '../providers'
 

